src/vscode-cosmosdb.api.d.ts

```diff
@@ -51,6 +51,8 @@ export interface DatabaseAccountTreeItem extends CosmosDBTreeItem {
         masterKey: string;
         documentEndpoint: string;
     }
+
+    openInPortal(resourceID: string): Promise<void>;
```

Comment:
How about we add `accountId` to `azureData` and just let the app service repo take care of the actual openInPortal logic. I feel like that's a simpler and more generic solution, both important qualities for an API

Reply:
This might be better option. I will just add the accountId in Databases and openInPortal will be through app service.